test/unit/foreman_google/google_compute_adapter_test.rb

```diff
@@ -113,7 +113,7 @@ class GoogleComputeAdapterTest < GoogleTestCase
 
         args = {
           name: 'foreman-test',
-          machine_type: 'zones/us-east1-b/machineTypes/e2-micro',
```

Comment:
This is correct, see https://github.com/stejskalleos/foreman_google/blob/develop/app/models/foreman_google/google_compute.rb#L86

Reply:
oh right, I thouth these are args for google_compute initialize, my bad!